File: dotnet-adapters-shortest-paths-core/main/com.programmerare.shortestpaths/core/validation/GraphValidationException.java
/*
* Copyright (c) Tomas Johansson , http://www.programmerare.com
* The code is made available under the terms of the MIT License.
* https://github.com/TomasJohansson/adapters-shortest-paths/blob/master/adapters-shortest-paths-core/License.txt
*/
package com.programmerare.shortestpaths.core.validation;

/**
 * @see GraphEdgesValidator
 *    
 * @author Tomas Johansson
*/
public class GraphValidationException extends RuntimeException {
	public GraphValidationException(String message) {
		super(message);
	}
}

File: Programmerare.ShortestPaths.Core/Programmerare.ShortestPaths/Core/validation/GraphValidationException.cs
/*
* Copyright (c) Tomas Johansson , http://www.programmerare.com
* The code in this "core" project is licensed with MIT.
* Other projects within this Visual Studio solution may be released with other licenses e.g. Apache.
* Please find more information in the files "License.txt" and "NOTICE.txt" 
* in the project root directory and/or in the solution root directory.
* It should also be possible to find more license information at this URL:
* https://github.com/TomasJohansson/adapters-shortest-paths-dotnet/
*/
using System;

namespace Programmerare.ShortestPaths.Core.Validation
{
    /// <summary>
    /// See <see cref="GraphEdgesValidator"/>
    /// </summary>
    public class GraphValidationException : Exception {
	    public GraphValidationException(string message): base(message) {
	    }
    }
}